bool warn
negomo_enshu/NegomoTask[] tasks
---
bool status
negomo_enshu/NegomoTask task